src/node_crypto.h

```diff
@@ -55,6 +55,13 @@
 # define NODE__HAVE_TLSEXT_STATUS_CB
 #endif  // !defined(OPENSSL_NO_TLSEXT) && defined(SSL_CTX_set_tlsext_status_cb)
 
+// TLS-PSK support requires OpenSSL v1.0.0 or later built with PSK enabled
+#if OPENSSL_VERSION_NUMBER >= 0x10000000L
+#ifndef OPENSSL_NO_PSK
+#define OPENSSL_PSK_SUPPORT
+#endif
+#endif
```

Comment:
This can probably go because we don't support openssl < 1.0.2.  I'd change the guards to `#ifndef  OPENSSL_NO_PSK`.

Reply:
I'm not sure I understand which guards I'd be updating. The one you referenced is already `#ifndef OPENSSL_NO_PSK`